sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2264,6 +2264,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val TRANSFORMATION_EXIT_TIMEOUT = buildConf("spark.sql.transformation.exitTimeoutInSeconds")
```

Comment:
`transformation` -> `scriptTransformation` to be more precise.

Reply:
done